src/Commands/AbstractDrupalCommands.php

```diff
@@ -316,8 +316,57 @@ public function drushSetup(array $options = [
     }
 
     /**
-     * Setup Drupal settings overrides.
+     * Setup Drupal services file.
+     *
+     * This command will copy the content of the file given in service-parameters to "services.yml" file in
+     *  the site directory.
+     *
+     * The services file name can be set in the Task Runner
+     * configuration by setting the "drupal.service_parameters" property.
+     *
+     * @command drupal:services-setup
+     *
+     * @option service-parameters       Drupal services filename.
+     * @option root                     Drupal root.
+     * @option sites-subdir             Drupal site subdirectory.
+     * @option force                    Drupal force generation of a new settings.php.
+     * @option skip-permissions-setup   Drupal skip permissions setup.
      *
+     * @param array $options
+     *
+     * @return \Robo\Collection\CollectionBuilder
+     */
```

Comment:
Should add ` * @throws \Exception`

Reply:
We don't do this in a consistent manner. Here I don't see a good reason to use it.